test/api/exclude-members.test.js

```diff
@@ -0,0 +1,70 @@
+const test = require('tape')
+const { promisify: p } = require('util')
+const { Server, replicate, FeedId } = require('../helpers')
+
+test('tribes.excludeMembers',async t => {
+  const kaitiaki = Server()
+  const newPerson = Server()
+
+  const name = id => {
+    if (id === kaitiaki.id) return 'kaitiaki '
+    if (id === newPerson.id) return 'new person'
+  }
+
+  try {
+    const { groupId, groupInitMsg } = await p(kaitiaki.tribes.create)({})
+    t.true(groupId, 'creates group')
+
+    const authorIds = [
+      newPerson.id,
+      FeedId()
+    ]
+
+    await p(kaitiaki.tribes.invite)(
+      groupId, authorIds, {}
+    )
+
+    await p(replicate)({ from: kaitiaki, to: newPerson, live: false, name })
+
+    let exclude = await p(kaitiaki.tribes.excludeMembers)(groupId, authorIds).catch(err=>t.error(err, err.message))
+
+    exclude = await p(kaitiaki.get)({ id: exclude.key, private: true })
+    const expected = {
+      type: 'group/exclude-member',
+      excludes: authorIds,
+
+      tangles: {
+        members: { root: groupInitMsg.key, previous: [groupInitMsg.key] },
+        group: { root: groupInitMsg.key, previous: [...exclude.content.tangles.group.previous] }
```

Comment:
is this cheating?

Reply:
i would say no.

buuut, there seemed to be two entries in the array, which is weird

Reply:
:hot_pepper: should be same as `tangles.members.previous`  I think

Reply:
mm yeah but problem is the members tangle doesn't work yet. there was another review comment about this, i'll make an issue

Reply:
https://github.com/ssbc/ssb-tribes/issues/76

Reply:
putting this as a later task if that's fine